extensions/graphite-emitter/src/main/java/io/druid/emitter/graphite/NoopGraphiteEventConverter.java

```diff
@@ -0,0 +1,129 @@
+package io.druid.emitter.graphite;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.common.base.Joiner;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSortedSet;
+import com.metamx.emitter.service.ServiceMetricEvent;
+
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Pattern;
+
+/**
+ * Emits all the events instance of {@link com.metamx.emitter.service.ServiceMetricEvent}.
+ * <p/>
+ * All the dimensions will be retained and lexicographically order using dimensions name.
+ * <p/>
+ * The metric path of the graphite event is:
+ * <namespacePrefix>.[<druid service name>].[<druid hostname>].<dimensions values ordered by dimension's name>.<metric>
+ * <p/>
+ * Note that this path will be sanitized by replacing all the `.` or `space` to `_` {@link NoopGraphiteEventConverter#sanitize(String)}
```

Comment:
you are only sanitizing the namespace prefix and not the entire metric path,so update the comment accordingly